pyvista/__init__.py

```diff
@@ -1,30 +1,30 @@
 """PyVista package for 3D plotting and mesh analysis."""
+# flake8: noqa: F401
```

Comment:
I agree with this policy, which is explicitly stated in the comments, rather than excluding files from the flake8 configuration.

Reply:
> Star imports are still used in a few intentional places but have otherwise been reduced.

Also, it is good for reducing star imports.